.gitlab-ci.yml

```diff
@@ -308,3 +308,28 @@ pages:
   only:
     - master
 
+package:appimage-client:
+  stage: package
+  image: appimagecrafters/appimage-builder
+  dependencies:
+    - build:ubuntu-16.04
+  before_script:
+    - apt-get update -y
+    - apt-get install -y git wget rsync
+    # Collect files
+    - mkdir AppDir
+    - rsync -a artifact/minetest/usr/ AppDir/usr/
```

Comment:
you could use `cp -a` here

Reply:
Right, didn't knew how to do that correctly. Google said rsync :D